packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -277,6 +277,7 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
           className={this._classNames.closeButton}
           onClick={this._onPanelClick}
           ariaLabel={closeButtonAriaLabel}
+          title={closeButtonAriaLabel}
```

Comment:
I'm uncertain we should do this by default. I think the guidance from W3 ARIA is to not use both `aria-label` and `title`. Perhaps we can make this opt-in. 

https://github.com/nvaccess/nvda/issues/7841#issuecomment-358644281

>The last resort is to use text from a tooltip attribute (such as the title attribute in HTML). This is used only if nothing else, including subtree content, has provided results.

Having both may cause assistive technology to double-read.

Reply:
>Having both may cause assistive technology to double-read. I haven't tried this evening admittedly.

Hey @KevinTCoughlin  i have tried it, screen reader is not reading it double time!

Reply:
Good to hear 👍. As long as Keros doesn't flag it as a violation then we're good. We might want to add a snapshot test also.

I think Keros as part of CI is currently disabled

Reply:
I tested this locally in Edge + Narrator and @shivasai09 is correct, the attribute does not result in a double read.